jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -121,14 +121,14 @@ preferences.decompile=Dekompilierung
 preferences.project=Projekt
 preferences.other=Andere
 preferences.language=Sprache
-preferences.lineNumbersMode=Modus zur Anzeige der Zeilennummern im Editor
+preferences.lineNumbersMode=Editor Zeilennummern-Modus
 preferences.check_for_updates=Nach Updates beim Start suchen
 preferences.fallback=Zwischencode ausgeben (einfacher Speicherauszug)
 preferences.showInconsistentCode=Inkonsistenten Code anzeigen
 preferences.escapeUnicode=Unicodezeichen escapen
 preferences.replaceConsts=Konstanten ersetzen
 preferences.respectBytecodeAccessModifiers=Modifikatoren für Bytecode-Zugriff beachten
-preferences.useImports=Generiere Import statements
+preferences.useImports=Importanweisungen anwenden
```

Comment:
This translation is in my opinion not better than before. Based on your description it is unclear what it means.